<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="DESKTOP-2CTJ19M" failures="1" tests="5" name="Personal Load" time="0.139" errors="0" timestamp="2022-08-15T00:04:10 CEST">
  <testcase classname="test.testType1" name="Demo" time="0.02">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at test.testType1.Demo(testType1.java:14)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- Demo -->
  <testcase classname="test.testType2" name="ploan" time="0.003"/>
  <testcase classname="test.testType4" name="LoginAPIHomeLoan" time="0.003"/>
  <testcase classname="test.testType4" name="MobileLoginHomeLoan" time="0.002"/>
  <testcase classname="test.testType4" name="WebloginHomeLoan" time="0.003"/>
  <testcase classname="test.testType1" name="secondTest" time="0.0">
    <ignored/>
  </testcase> <!-- secondTest -->
</testsuite> <!-- Personal Load -->
